tests/kernel/mutex/sys_mutex/src/thread_competition.c

```diff
@@ -7,10 +7,6 @@
 #include <ztest.h>
 #include <sys/mutex.h>
 
-#define HIGH_T1	 0xaaa
-#define HIGH_T2	 0xbbb
-#define LOW_PRO	 0xccc
```

Comment:
I don't understand.  The old version had named constants for your flag cookies, but the new one is hard-coding the values?  That seems to be going backwards.

Reply:
Hi @andyross , I totally agree with you, and I have update this.